booking/manager.py

```diff
@@ -0,0 +1,47 @@
+from booking.storage import get_storage
+from booking.models import Event, TicketOrder
+
+
+class DataManager:
+
+    def __init__(self):
+        self._storage = get_storage()
+
+    def can_buy_tickets(self, event_id):
+        orders = self._storage.get_orders_for_event(event_id)
+        active_orders = filter(lambda order: order.activate, orders)
+        tickets_sold = sum(order.ticket_count for order in active_orders)
+        total_tickets = self._storage._events.get(event_id).total_ticket_count
+        return total_tickets - tickets_sold
```

Comment:
Any number different than `0` will be interpreted as boolean `True` so the check if the value is `> 0` is necessary.
```suggestion
        return (total_tickets - tickets_sold) >= ticket_count
```